client/my-sites/google-my-business/new-account/index.js

```diff
@@ -64,10 +64,10 @@ class GoogleMyBusinessNewAccount extends Component {
 			<Main className="gmb-new-account" wideLayout>
 				<PageViewTracker path="/google-my-business/new/:site" title="Google My Business > New" />
```

Comment:
```suggestion
				<PageViewTracker path="/google-my-business/new/:site" title="Google Business Profile > New" />
```

I think we are fine to refactor the `PageViewTracker` title. The last I checked those paths are rarely viewed anyway.

Reply:
All done - thanks for taking a look! :)